applications/ContactStructuralMechanicsApplication/custom_processes/find_intersected_geometrical_objects_with_obb_for_contact_search_process.h

```diff
@@ -107,7 +107,7 @@ class KRATOS_API(CONTACT_STRUCTURAL_MECHANICS_APPLICATION) FindIntersectedGeomet
             BaseProcessType::INTERSECTING_ELEMENTS|
             BaseProcessType::INTERSECTED_CONDITIONS|
             BaseProcessType::INTERSECTED_ELEMENTS|
-            ~BaseType::DEBUG_OBB|
+            BaseType::DEBUG_OBB.AsFalse()|
```

Comment:
@rubenzorrilla why did this even compile?
Shouldn't this operator be removed?
Or is it still used somewhere else?

Reply:
The point is that the operator `~` is not the same as our flavour of the `False`. This is why we keep it.